regcore/db/django_models.py

```diff
@@ -66,6 +66,8 @@ def _serialize(self, reg, adjacency_map):
                 for child in adjacency_map.get(reg.id, [])
             ],
         }
+        if reg.lft:
```

Comment:
From the implementation of the algorithm, it cannot be as it starts numbering at 1. But it is better to be safe and not rely on an implementation.